src/js/health-records/utils/validations.js

```diff
@@ -0,0 +1,6 @@
+export function isValidDateRange(startDate, endDate) {
+  if (!startDate || !endDate) {
+    return true;
```

Comment:
I think you originally wrote this function, but it made sense to me. If either is null, there's no need to check if it's a valid range since there's no comparison to be made.